dial-cookbook/examples/how_to_call_image_to_text_applications.ipynb

```diff
@@ -617,8 +617,8 @@
     "  llm.generate(messages=[[HumanMessage(content=\"\", additional_kwargs=extra_fields)]])\n",
     "\n",
     "  raise Exception(\"Generation didn't fail\")\n",
-    "except Exception as e:\n",
-    "  assert str(e) == \"Error code: 422 - {'error': {'message': 'No image attachment was found in the last message', 'type': 'runtime_error'}}\", \"Unexpected error\""
+    "except openai.UnprocessableEntityError as e:\n",
+    "  assert e.body[\"message\"] == \"No image attachment was found in the last message\""
```

Comment:
I'm not sure that's important here, but body parameter can be None in this (error)[https://github.com/openai/openai-python/blob/9850c169c4126fd04dc6796e4685f1b9e4924aa4/src/openai/_exceptions.py#L35]

Reply:
I agree, but let's not overcomplicate the code, which is only used for a demonstation of a fixed scenario with an expected outcome.